src/html/common.css

```diff
@@ -0,0 +1,333 @@
+/* External CSS style file */
```

Comment:
Please refactor all CSS into Styled Components. You can find some more information here: https://redback-operations.github.io/redback-documentation/docs/web-mobile-app-dev/frontend/styled-components